replication-api-impl/src/main/java/com/connexta/replication/api/impl/persistence/pojo/SitePojo.java

```diff
@@ -99,74 +137,141 @@ public SitePojo setUrl(@Nullable String url) {
   }
 
   /**
-   * When {@code false}, the local process is responsible for running replicator configs associated
-   * with this site.
+   * Gets the type of this site.
+   *
+   * @return the type for this site
+   */
+  @Nullable
+  public String getType() {
+    return type;
+  }
+
+  /**
+   * Sets the type of this site.
    *
-   * <p>When {@code true}, the local process is no longer responsible for performing replication for
-   * any replicator configs associated with this site. This effectively disables running replication
-   * locally.
+   * @param type the type for the site
+   * @return this for chaining
+   */
+  public SitePojo setType(@Nullable String type) {
+    this.type = type;
+    return this;
+  }
+
+  /**
+   * Gets the kind of this site.
    *
-   * @return {@code true} if replication should not run locally, otherwise {@code false}.
+   * @return the kind for this site
    */
-  public boolean isRemoteManaged() {
-    return isRemoteManaged;
+  @Nullable
+  public String getKind() {
+    return kind;
   }
 
   /**
-   * See {@link #isRemoteManaged()}.
+   * Sets the kind of this site.
    *
-   * @param remoteManaged whether or not the local process is responsible for running replications
-   *     this site is associated with.
+   * @param kind the kind for the site
    * @return this for chaining
    */
-  public SitePojo setRemoteManaged(boolean remoteManaged) {
-    this.isRemoteManaged = remoteManaged;
+  public SitePojo setKind(@Nullable String kind) {
+    this.kind = kind;
     return this;
   }
 
   /**
-   * Gets the type of this site.
+   * Gets the optional amount of time in milliseconds to wait in between polling attempts whenever
+   * polling for intel from this site.
    *
-   * @return the type for this site
+   * @return the maximum amount of time in milliseconds to wait in between polling attempts or
+   *     <code>0L</code> if polling is not required or if the local configured default value should
+   *     be used
    */
   @Nullable
-  public String getType() {
-    return type;
+  public long getPollingTimeout() {
+    return pollingTimeout;
   }
 
   /**
-   * Sets the type of this site.
+   * Sets the optional amount of time in milliseconds to wait in between polling attempts whenever
+   * polling for intel from this site.
    *
-   * @param type the type for the site
+   * @param pollingTimeout the maximum amount of time in milliseconds to wait in between polling
+   *     attempts or <code>0L</code> if polling is not required or if the local configured default
+   *     value should be used
    * @return this for chaining
    */
-  public SitePojo setType(@Nullable String type) {
-    this.type = type;
+  public SitePojo setPollingTimeout(long pollingTimeout) {
+    this.pollingTimeout = pollingTimeout;
+    return this;
+  }
+
+  /**
+   * Gets the parallelism factor for this site. This corresponds to the maximum number of pieces of
```

Comment:
Not a fan of jumping left and right. Fact is that these are separate classes which can evolve differently. Already some parameters are not the same between some of them since they have to be converted.